tcf_website/tests/test_utils.py

```diff
@@ -202,3 +290,12 @@ def new_function(*args, **kwargs):
         logger.setLevel(previous_logging_level)
 
     return new_function
+
+
+def create_new_sem(obj, year):
```

Comment:
I looked through all the diffs in this PR, but this seems like the only case where an abbreviation is used. I don't think adding `ester` at the end will hurt.